core/schemas/__init__.py

```diff
@@ -0,0 +1,120 @@
+import importlib
+import inspect
+import logging
+from pathlib import Path
+
+import aenum
+
+from core.schemas import entity, indicator, observable
+
+logger = logging.getLogger(__name__)
+
+
+def load_entities():
```

Comment:
Loading indicators, entities, and observables is done in sensibly the same way. Can we factor this into a function with parameters to stay DRY?

Reply:
Agree. What about something like:

```python
for schema in ["observable", "entity", "indicator"]:
    load_schemas(schema)
```

Reply:
Looks good. If the lines too long feel free to extract that list into a global

Reply:
Actually, trying to bring the same logic into only one function becomes cryptic. Furthermore, each type of schemas has their own tweaks and makes it hard to generalize.